server/interview/interview.model.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+let InterviewSchema = new Schema({
+    participants: [{ type: Schema.Types.ObjectId, ref: 'User' }],
+    questions: [{ type: Schema.Types.ObjectId, ref: 'Question' }]
```

Comment:
I think the interview schema should also have the difficulty level - it's okay to have duplicated information in a NoSQL database

Reply:
Added difficulty to interview